content/credits.md

```diff
@@ -3,7 +3,7 @@ title: Strathearn Geeks — Credits
 description: The Strathearn Geeks group is a meetup for makers, coders, designers, 3D printers and all tech-minded folks in Crieff and the surrounding areas. Visit our website for more information.
 ---
 
-Site design by Strathearn Geeks, provided under the terms of the GPL3. The Strathearn Geeks site uses the following libraries and resources —
+Site design by Strathearn Geeks, provided under the terms of the CC BY-SA 4.0. The Strathearn Geeks site uses the following libraries and resources —
```

Comment:
```suggestion
Site design by Strathearn Geeks, content provided under the terms of the CC BY-SA 4.0. The Strathearn Geeks site uses the following libraries and resources —
```

I'd actually be in favour of you adding the theme as a resource here and crediting yourself in creating it if you want to...

Reply:
Done :)